graylog2-server/src/main/java/org/graylog2/initializers/IndexerSetupService.java

```diff
@@ -114,11 +111,10 @@ protected void startUp() throws Exception {
         Tools.silenceUncaughtExceptionsInThisThread();
 
         LOG.debug("Starting indexer");
-        try {
-            node.start();
+        node.start();
```

Comment:
The behavior is the same as before as the outer most `catch` block just re-threw the exception: https://github.com/Graylog2/graylog2-server/pull/1289/files#diff-72fd860130aebd46f43cb3fa1fb32d18L193